### ISSUE TITLE
Two semicolon in css files

### ISSUE THREAD

**Issue by VictorKolb** [zoobestik/csso-webpack-plugin#15]
Two semicolon in css files breaks plugin

```
.sharebox {
  padding: 24px 0 36px 0;;←
}
```

This give a error:
```
ERROR in style.css from CssoWebpackPlugin
Cannot read property '0' of undefined TypeError: Cannot read property '0' of undefined
```

I searched it about two hours 😬

**Comment by zoobestik** [comment]
@VictorKolb, thank you for your report, I'm agree error message should be better.

But it's responsibility of [`csso`](https://github.com/css/csso) (aka **csstree**) :
```
Cannot read property '0' of undefined TypeError: Cannot read property '0' of undefined
    at getPropertyDescriptor (.../css-tree/lib/utils/names.js:65:24)
    at getPropertyFingerprint (.../csso/lib/restructure/6-restructBlock.js:68:20)
    at List.<anonymous> (.../csso/lib/restructure/6-restructBlock.js:213:27)
    at List.eachRight (.../css-tree/lib/utils/list.js:182:12)
    at Object.processRule (.../csso/lib/restructure/6-restructBlock.js:211:18)
```

and impossible to improve in this plugin.

You should create issue for original repository or vote something already exist like https://github.com/css/csso/issues/370

**UPD**: https://github.com/css/csso/issues/378

**Comment by lahmatiy** [comment]
The issue was fixed in CSSO 4.0.2.
